skillmap/src/App.tsx

```diff
@@ -354,11 +359,11 @@ class AppImpl extends React.Component<AppProps, AppState> {
 
     render() {
         const { skillMaps, activityOpen, backgroundImageUrl, theme } = this.props;
-        const { error, syncingLocalState } = this.state;
+        const { error, showingSyncLoader } = this.state;
         const maps = Object.keys(skillMaps).map((id: string) => skillMaps[id]);
         return (<div className={`app-container ${pxt.appTarget.id}`}>
                 <HeaderBar />
-                {syncingLocalState && <div className={"makecode-frame-loader"}>
+                {showingSyncLoader && <div className={"makecode-frame-loader"}>
```

Comment:
do we even need `syncingLocalState` any more? could we remove it entirely?

Reply:
nope, removed!